usersService/src/main/java/il/ac/afeka/usersservice/api/ReactiveUsersController.java

```diff
@@ -0,0 +1,68 @@
+package il.ac.afeka.usersservice.api;
+
+import il.ac.afeka.usersservice.logic.UsersService;
+import il.ac.afeka.usersservice.boundaries.UserBoundary;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import org.springframework.web.server.ResponseStatusException;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+@RestController
+@RequestMapping(path = {"/users"})
+public class ReactiveUsersController {
+	private UsersService usersService;
+
+	public ReactiveUsersController(UsersService usersService) {
+		this.usersService = usersService;
+	}
+
+	// Triggered when no parameters are provided
+	@GetMapping(produces = {MediaType.TEXT_EVENT_STREAM_VALUE})
+	public Flux<UserBoundary> getAllUsers() {
+		return this.usersService.getAll().log();
+	}
+
+	@PostMapping(
+			consumes = {MediaType.APPLICATION_JSON_VALUE},
+			produces = {MediaType.APPLICATION_JSON_VALUE})
+	public Mono<UserBoundary> createUser(@RequestBody UserBoundary user) {
```

Comment:
```suggestion
	public Mono<UserBoundary> createUser(@RequestBody NewUserBoundary user) {
```

this is my bad, but should be fixed here as well.
when user is provided from the outside it should provide a password, which is not present in the current `UserBoundary`.